CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to osbuild-composer
+
+First of all, thank you for taking the time to contribute to osbuild-composer. In this document you will find information that can help you with your contribution.
+
+### Running from sources
+
+We recommend using the latest stable Fedora for development but latest minor release of RHEL/CentOS 8 should work just fine as well. To run osbuild-composer from sources, follow these steps:
+
+```
+$ git clone https://github.com/osbuild/osbuild-composer.git
+$ cd osbuild-composer
+$ dnf group install 'RPM Development Tools' # Install rpmbuild
+$ dnf builddep golang-github-osbuild-composer.spec # Install build-time dependencies
+$ make rpm
+$ sudo dnf install rpmbuild/RPMS/x86_64/* -y
+$ sudo dnf install cockpit-composer -y
+$ sudo systemctl enable --now osbuild-composer.socket
+$ sudo systemctl enable --now cockpit.socket
+```
```

Comment:
Can we split this into system-setup, repo-setup, and then the recurring build-run-cycle? Something like this:



Clone the repository and create a local checkout:

```
$ git clone https://github.com/osbuild/osbuild-composer.git
$ cd osbuild-composer
```

To install the build-requirements for Fedora and friends, use:

```
$ dnf group install 'RPM Development Tools' # Install rpmbuild
$ dnf builddep golang-github-osbuild-composer.spec # Install build-time dependencies
$ sudo dnf -y install cockpit-composer # Optional: Install cockpit integration
$ sudo systemctl start cockpit.socket # Optional: Start cockpit
```

Finally, use the following to compile the project from the working directory, install it, and then run it:

```
$ make rpm
$ sudo dnf -y install rpmbuild/RPMS/x86_64/*
$ sudo systemctl start osbuild-composer.socket
```

Reply:
sounds good, thanks!